build_versions.yaml

```diff
@@ -7,3 +7,4 @@ include:
     db_version: "8.9.1-1+ubuntu24.04"
     r10k_version: "5.0.0"
     rugged_version: "1.9.0"
+    jdk_version: "17"
```

Comment:
in case you want to go to latest LTS: JDK21 passes our tests on openvoxserver as well.